environments/production/elasticsearch.yml

```
@@ -0,0 +1,3 @@
+settings:
+  case_search:
+    number_of_shards: 16
```

From discussion in https://docs.google.com/document/d/1W34WJZ4rhwflRSEtgH_kJZXFyTnr-fg1sXDn-kHUY9k/edit?disco=AAAAIq2ke4A and slack
```suggestion
    number_of_shards: 40
```

---

Never mind, I talked myself down from this. I'm resolving